core/src/Streamly/Internal/Data/Stream/Expand.hs

```diff
@@ -186,10 +187,15 @@ infixr 6 `append`
 --
 -- /Pre-release/
 --
-{-# INLINE append #-}
-append :: Stream m a -> Stream m a -> Stream m a
+{-# INLINE_NORMAL append #-}
+append :: Monad m => Stream m a -- ^ Monad contraint due to fuse re write RULES
```

Comment:
This comment applies to "Stream m a", haddock will show it for that which would be wrong.